static/app/utils/cursor.tsx

```diff
@@ -0,0 +1,26 @@
+export type CursorInfo = {
+  isPrev: boolean;
+  offset: number;
+  value: number;
+};
```

Comment:
Is there the need to store all of this information? Looks like you only need the `offset`.

Reply:
This is a generic function that parses a cursor ( that's why I put it in app/utils/cursor.tsx), hence I extract all the information in the Cursor for anybody to use.

I only use the offset, if I had it as a local util I would have only looked for the offset but I decided it is a potentially useful util (since we have cursors and pagination all over the UI, and we always seem to parse it locally).